checksetup.pl

```diff
@@ -279,12 +279,6 @@ BEGIN
     # Reset the mod_perl pre-load list
     unlink(Bugzilla::Constants::bz_locations()->{datadir} . '/mod_perl_preload');
 
-    # Check if the default parameter for urlbase is still set, and if so, give
-    # notification that they should go and visit editparams.cgi
-    if (Bugzilla->params->{'urlbase'} eq '') {
-        print "\n" . get_text('install_urlbase_default') . "\n"
-            unless $silent;
-    }
```

Comment:
probably better to leave this warning in place if Bugzilla->localconfig->{urlbase} is not set.

Reply:
you wanted me to remind you to check that urlbase ends in a /.
maybe also validate that urlbase is a url, with "http" or "https" as the scheme.